.github/workflows/functional-test-cloud.yaml

```diff
@@ -314,6 +314,64 @@ jobs:
           append: true
           message: |
             :hourglass: Publishing Bicep Recipes for functional tests...
+      
```

Comment:
Moving this into the build job so we make sure to upload any bicep changes before pushing recipes